flake.lock

```diff
@@ -20,11 +20,11 @@
     "blazesym": {
       "flake": false,
       "locked": {
-        "lastModified": 1742330784,
-        "narHash": "sha256-bnXRy6Me+5bzUB/1i53qLCbAkvrAA/Ihxm24el/906Y=",
+        "lastModified": 1742522152,
```

Comment:
looks like blazesym is being updated as well. is that intentional? instructions for how to update a single dep available here: https://github.com/bpftrace/bpftrace/blob/master/docs/nix.md#update-flake-inputs

Reply:
Right, I fixed the PR to update only nixpkgs.